### DISCUSSION TITLE
Merge conflicts in PR sent by mergify

### DISCUSSION THREAD

**Discussion by mdelapenya** [Mergifyio/mergify#2466] [Ideas]
Hi folks, first of all, thanks for this tool, we are starting to use it more and more, and seeing a good future for it.

I'd like to raise a question related to the PRs sent by @mergifyio bot when there are conflicts. TBH I'm not used to seeing PRs with merge conflict being submitted, so for me it's weird to see them in the open PRs list for a project. It could be the case there is an automerge rule 🤷‍♀️ , or a human committing a mistake merging it... or unnecessary pings to persons/teams for broken code.

I'd prefer seeing a message in the original PR saying something like _"sorry but we cannot backport this PR to $BRANCH because of merge conflicts. Please check the $BRANCH branch and try it again"_.

**Comment by jd** [comment]
You can set `ignore_conflicts` to `False` here https://docs.mergify.io/actions/backport/#options

That'll make sure no PR is created in case of conflicts.

**Reply by mdelapenya** [reply]
Oh thanks!! This is the RTFM issue... My bad Thanks for pointing out to the docs.

![](https://cdn.memegenerator.es/imagenes/memes/thumb/4/88/4887789.jpg)
